src/transformers/pipelines/image_classification.py

```diff
@@ -1,3 +1,4 @@
+import warnings
```

Comment:
This file is lacking the copyright! 
Can you add it?

Reply:
Done!